test/itg/1.7/scenario/BO/install_and_uninstall_module.js

```diff
@@ -60,23 +61,35 @@ describe('The Install of a Module and its Uninstall', function () {
             this.client
                 .setValue(this.selector.modules_search, module_tech_name)
                 .click(this.selector.modules_search_button)
-                .waitForExist(this.selector.module_tech_name, 90000)
-                .call(done);
+                .getText(this.selector.nbr_module).then(function (text) {
+                global.nbr = text[0];
```

Comment:
This is the number of modules returned by the search function. It has to be a number instead of text. 
I will fix it.

 Thank you @Quetzacoalt91 !